src/icons.yml

```diff
@@ -3958,6 +3958,7 @@ icons:
     aliases:
       - institution
       - bank
+      - government
```

Comment:
just add a `filter:` line before this line

filter should be 4 space indented, like `aliases:`